public/components/ContentPanel/index.scss

```diff
@@ -14,12 +14,11 @@
  */
 
 .content-panel-title {
-  color: #3f3f3f;
+  color: #000;
```

Comment:
How about remove this color, we can use the default color to keep consistent.

Reply:
sure. will remove `content-panel-title` class as it only has `color` attribute.